pkg/cloudprovider/providers/openstack/openstack_volumes.go

```diff
@@ -193,6 +195,36 @@ func (volumes *VolumesV2) deleteVolume(volumeID string) error {
 	return err
 }
 
+func (volumes *VolumesV1) expandVolume(volumeID string, newSize int) error {
+	startTime := time.Now()
+	create_opts := volumeexpand.ExtendSizeOpts{
+		NewSize: newSize,
+	}
+	err := volumeexpand.ExtendSize(volumes.blockstorage, volumeID, create_opts).ExtractErr()
+	timeTaken := time.Since(startTime).Seconds()
+	recordOpenstackOperationMetric("expand_volume", timeTaken, err)
+	if err != nil {
+		glog.Errorf("Cannot expand volume %s: %v", volumeID, err)
```

Comment:
Don't log-and-return errors, just return them.

Reply:
Yeah, i will delete the log

Reply:
done